man/g.report.part5.Rd

```diff
@@ -48,6 +50,16 @@
     Inclusion criteria for number of valid hours, either
     as expressed as a ratio of 1 or as the number of hours in a 24 hour day.
   }
+  \item{segmentWEARcrit.part5}{
+    Inclusion criteria for number of valid hours in the segments defined with
+    qwindow in part 5, expressed as a ratio of 1.
```

Comment:
I am struggling to follow this sentence. It refers to number of valid hours but it does not take number valid hours as input., and it does not clarify what 0 or 1 stands for. Does 1 mean fully valid or does it mean fully invalid?

Rephrase as:
Fraction of \code{qwindow} segments that needs to valid in part 5, where 0.3 means indicates that at least 30 percent of the time should be valid.